.clang-format

```diff
@@ -0,0 +1,65 @@
+---
+# Language
+Language:        Cpp
+Standard:        Cpp11 # newer ones are not supported currently
```

Comment:
We now use C++14.

Reply:
I wrote the comment because all other don't seem to work with current versions of clang-format (11.0.0).

Reply:
From https://releases.llvm.org/11.0.0/tools/clang/docs/ClangFormatStyleOptions.html:
> `Cpp11` is a deprecated alias for `Latest`

This means we should use `c++17` since we again moved to C++17.